docs/rules/no-domains.md

```diff
@@ -0,0 +1,37 @@
+# No Domains
+
+Rule `no-domains` will enforce that domains are not used within urls.
```

Comment:
I'm a fan of consistency.  I also don't mind having the two rules right next to each other in the file system. Updated.